src/main/java/com/airlines/dao/impl/AirplaneDaoImpl.java

```diff
@@ -33,29 +33,29 @@ public AirplaneDaoImpl(DataSource dataSource) {
     public void save(Airplane airplane) {
         try (Connection connection = dataSource.getConnection();
              PreparedStatement insertStatement = connection.prepareStatement(INSERT_QUERY,
-                     PreparedStatement.RETURN_GENERATED_KEYS);) {
+                     PreparedStatement.RETURN_GENERATED_KEYS)) {
             insertStatement.setString(1, airplane.getCodeName());
             insertStatement.setString(2, airplane.getModel());
             insertStatement.setDate(3, Date.valueOf(airplane.getManufactureDate()));
             insertStatement.setInt(4, airplane.getCapacity());
             insertStatement.setInt(5, airplane.getFlightRange());
-            insertStatement.setObject(6, airplane.getCrew());
+            insertStatement.setObject(6, airplane.getCrewMember());
             insertStatement.executeUpdate();
             ResultSet generatedKeys = insertStatement.getGeneratedKeys();
             if (generatedKeys.next()) {
                 long id = generatedKeys.getLong(1);
-                airplane.setId(id);
+                Airplane.builder().withId(id);
```

Comment:
Do same changes as for CrewDaoImpl : (copy attributes and set id, change signature)